frontend/src/static/js/v_zoom.js

```diff
@@ -6,17 +11,24 @@ window.vZoom = {
       filterTimeFrame: window.hashParams.timeframe,
       showAllCommitMessageBody: true,
       expandedCommitMessagesCount: this.totalCommitMessageBodyCount,
+      commitsSortType: 'time',
+      toReverseSortedCommits: true,
     };
   },
+
   computed: {
+    sortingFunction() {
+      return (a, b) => (this.toReverseSortedCommits ? -1 : 1)
+      * window.comparator(commitSortDict[this.commitsSortType])(a, b);
```

Comment:
Can we make this change for authorship too? Will help in maintaining consistency.

Reply:
It can be done in another PR though.

Reply:
Okay sure